syntax = "proto3";

message PlayerAction {
  enum Action {
		Idle = 0;

		StartMovingUp = 1;
		StartMovingRight = 2;
		StartMovingDown = 3;
		StartMovingLeft = 4;
		
		StopMovingUp = 5;
		StopMovingRight = 6;
		StopMovingDown = 7;
		StopMovingLeft = 8;
	}

	Action key_pressed = 1;
}
